TDD_RecentFileList/src/recentFileListPackage/_TestRecentFileList.java

```diff
@@ -0,0 +1,23 @@
+package recentFileListPackage;
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class _TestRecentFileList {
+
+	@Test
+	public void testFirstTime() {
```

Comment:
I think that is not necessary start the method's name with test we know that this is a test , but is necessary put a name to the test method that reflect the testing case and the method to be tested , we can talk about it